### ISSUE TITLE
some logic problem in MMVCServerSIO.py at line 197~203

### ISSUE THREAD

**Issue by pz9k** [w-okada/voice-changer#404]
### Issue Type

Bug Report

### vc client version number

1.5.3.8

### OS

Ubuntu

### GPU

3070

### Clear setting

no

### Sample model

no

### Input chunk num

no

### Wait for a while

no

### read tutorial

yes

### Voice Changer type

RVC

### Model type

pyTorch

### Situation

The code in server/MMVCServerSIO.py at line 197~203 is:
```
    else:  # 直接python起動
        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
        s.connect(("8.8.8.8", 80))
        hostname = s.getsockname()[0]
        if args.https == 1:
            printMessage(f"https://localhost:{PORT}/", level=1)
            printMessage(f"https://{hostname}:{PORT}/", level=1)
```
The hostname is only used while args.https == 1, I think the code should be like:
```
    else:  # 直接python起動
        if args.https == 1:
            s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
            s.connect(("8.8.8.8", 80))
            hostname = s.getsockname()[0]
            printMessage(f"https://localhost:{PORT}/", level=1)
            printMessage(f"https://{hostname}:{PORT}/", level=1)
```
Change the code will avoid some network errors when we are using VC-SERVER in the network which is not public or there are some DNS problems. (That's how I dicovered this error.)

**Comment by w-okada** [comment]
Which error can be avoided? I can not image them.

**Comment by pz9k** [comment]
The hostname in the code is only used while https=true.
Now, I set the https=false, in this case the hostname is never used, but the code will still try to get the hostname. 
Is it necessary to get the hostname when just using local http?
In some countries, sometimes it's hard to access 8.8.8.8.
And if the local DNS got errors, the PC can't reach 8.8.8.8,  and the exception will be thought out.

And when I am using two PCs(one as server) in a local area network which can't reach public Internet, the server PC can't reach 8.8.8.8, then I'll get the same exception.

After I modified the code, vc server launch successfully with bad network environment.
